web/src/containers/activity/NonPersonnelCost/index.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useState } from 'react';
+
+import Form from './NonPersonnelCostForm';
+import Review from './NonPersonnelCostReview';
+
+const NonPersonnelCost = ({ initialCollapsed, ...rest }) => {
+  const [collapsed, setCollapsed] = useState(initialCollapsed);
+  const collapse = useCallback(() => setCollapsed(true), []);
+  const expand = useCallback(() => setCollapsed(false), []);
+
+  if (collapsed) {
+    return <Review {...rest} expand={expand} />;
+  }
+  return <Form {...rest} collapse={collapse} />;
```

Comment:
Having the form and review broken out this way makes this so much easier to reason about! ![yay](https://media1.tenor.com/images/e47b4af417fbecdd7dc7290b107f8e99/tenor.gif?itemid=11572934)

Reply:
He looks so pained!